conductor/lib/nuke_utils.py

```diff
@@ -166,6 +166,17 @@ def get_nuke_script_filepath():
         raise Exception("Nuke script has not been saved to a file location.  Please save file before submitting to Conductor.")
     return filepath
 
+def check_script_modified():
+    '''
+    Check if the scene's been modified, and error out if it has been
+    '''
+    return nuke.root().modified()
+
+def save_current_nuke_script():
+    '''
+    Save the current script
+    '''
+    nuke.scriptSave()
```

Comment:
Can you return the value returned from `scriptSave`? https://learn.foundry.com/nuke/developers/70/pythonreference/

Reply:
done